src/commcare_cloud/environment/schemas/app_processes.py

```diff
@@ -137,7 +137,8 @@ def __new__(cls, name, required=True, is_queue=True, blockage_threshold=None,
     CeleryProcess("ucr_indicator_queue", required=False, blockage_threshold=60 * 60),
     CeleryProcess("ucr_queue", required=False, blockage_threshold=60 * 60),
     CeleryProcess("user_import_queue", required=False, blockage_threshold=60 * 60),
-    CeleryProcess("ush_background_tasks", required=False, blockage_threshold=3 * 60 * 60)
+    CeleryProcess("ush_background_tasks", required=False, blockage_threshold=3 * 60 * 60),
+    CeleryProcess("geospatial_queue", required=False, blockage_threshold=3 * 60 * 60)
```

Comment:
nit: Do you want to update this threshold considering we would have long running tasks?

Reply:
I was thinking of perhaps taking this up to 6 hours, however I'm not sure what the potential consequences are for raising the threshold. Then again, this queue will only be running the Geospatial indexing task, so I can't think there would be any major negative implications in raising this limit.

Tagging @gherceg to get his thoughts on whether raising this limit to 6 hours sounds fine.

Reply:
> I'm not sure what the potential consequences are for raising the threshold

I believe this is used to check if there is any issue with the queues. Considering we expect the queue to be blocked with one task for long, 3 hours of threshold for wait period for a new task seems low.

For this queue, this could probably be the time we expect a large data load to take.

Reply:
On the production envrionment, a task processing the max case limit takes around 1.5 hours. Given this, I feel a time limit of 3-4 hours should then be fine. 

The time taken is higher on smaller instances however (e.g. staging takes ~3.5 hours), so if we want to consider these instances as well then something closer to 4-6 hours would make more sense.

Reply:
Alright :+1:

Reply:
Blockage threshold has been increased to 6 hours ([01dfb8d](https://github.com/dimagi/commcare-cloud/pull/6430/commits/01dfb8d20ea6b7d09696753afc4dd5ed1d74506b)).